monkey/infection_monkey/exploit/win_ms08_067.py

```diff
@@ -241,8 +241,8 @@ def _exploit_host(self):
                       build_monkey_commandline(self.host, get_monkey_depth() - 1)
 
         try:
-            sock.send("start %s\r\n" % (cmdline,))
-            sock.send("net user %s /delete\r\n" % (self._config.user_to_add,))
```

Comment:
This change will require testing

Reply:
that it's binary? I did a standalone test. socket.send indeed receives binary digits and not string. Or what sort of test did you have in mind?

Reply:
Check the exploiter still works

Reply:
Problematic, MonkeyZoo doesn't have a test for this